---
layout: archive
title: "My research"
permalink: /research/
author_profile: true
---

My work lies at the intersection of bioinformatics, genomics, marine biology, and microbial ecology.
I use environmental genomics and transcriptomics to identify and quantify the metabolic potential of marine microorganisms. In particular,
I am interested in improving the efficiency of biofuel production and the sustainability of commercially grown algae using bioinformatic approaches.
Much of my work is funded from partnerships with companies interested in the commercial applications of my research.
 
In prior work, I [characterized an algae farm's most productive strain](/publications/AlgaeGenome2021) and
[simulated the reaction of a keystone crustacean to the potential impacts of climate change](/publications/DaphniaPulicaria2022). 
My current and future dissertation work is centered around algal polysaccharides and the microbes that degrade them, which have implications in
energy production, aquaculture, and marine ecosystem dynamics.  

Automated monitoring of algae farms
-----
In one research project, I have collaborated with the Hawaiian algaculture company [Global Algae](https://www.globalgae.com/)
to develop an system that monitors the microbiome of commercially grown algae. In the first stage of the project, I
assembled the genome of a commercial <i>Nitzschia inconspicua</i> strain and revealed 
the genetic factors that made this strain suitable for biofuel production. These genomic resources are publicly available at the Joint Genome Institute's
[PhycoCosm portal](https://phycocosm.jgi.doe.gov/Nithil2/Nithil2.home.html) to power future algal research.

In the second stage of this project, I am building a pipeline for the automated processing of metatranscriptomic data from algae ponds. This would enable
near real-time decision making regarding the evaluation of growth conditions and provide early warnings about bacterial and viral contamination. I am also
using metatranscriptomic data to quantify how the diatom microbiome responds to different nutrient levels and other stresses.

Enzymatic degradation of macroalgal polysaccharides
-----
My dissertation work focuses on the sugars inside seaweeds, which are notoriously difficult to break down using physical or chemical treatments. I use the gut of the
herbivorous [Hawaiian chub](https://www.marinelifephotography.com/fishes/chubs/kyphosus-hawaiiensis.htm) as a model system for studying the bacteria that degrade seaweeds in nature. This work is in parternship with the mariculture firm [Ocean Era](http://ocean-era.com/),
who are excited by the potential discovery of novel enzymes that can degrade macroalgae. If successful, this work would enable the integration of partially digested
seaweed compounds into commercial fish feed and improve the sustainability of aquaculture. Likewise, an more efficient method for algal decomposition improves
the viability of seaweed-based biofuel production.

Bioinformatics support
-----
Outside of my primary research projects, I have also collaborated with wet lab biologists to integrate computational approaches into their studies.
In one project with the lab of ecologist [Prof. Jon Shurin](https://shurinlab.biosci.ucsd.edu/), I served as the primary bioinformatician and paired
gene expression data with life history changes to explain the response of <i>Daphnia</i> crustaceans to rising temperatures and predation stress. In an ongoing collaboration with
molecular biologist [Prof. Stephen Mayfield](https://algae.ucsd.edu/mayfield/index.html) and his lab, I am leveraging my experience with algae genomics to catalogue
how the promoters of <i>Chlamydomonas</i> differ between species. I might also play a supportive role, as in one study where I supported research programmer [Dr. Sheila Podell](https://scholar.google.com/citations?user=YorpORcAAAAJ&hl=en)
in her discovery of potential [microbial contributions to a sponge's ability to degrade halogenated compounds](/publications/SpongeMicrobes2020).